openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -11045,16 +11052,31 @@ OS:HeatPump:WaterToWater:EquationFit:Heating,
    N13, \field Heating Compressor Power Coefficient 4
         \required-field
         \type real
-   N14; \field Heating Compressor Power Coefficient 5
+   N14, \field Heating Compressor Power Coefficient 5
         \required-field
         \type real
+   N15, \field Reference Coefficient of Performance
+        \note This optional field is used to autosize Reference Heating Power Consumption
+        \note COP = Reference Heating Capacity / Reference Heating Power Consumption
+        \type real
+        \minimum> 0.0
+        \default 7.5
+   N16, \field Sizing Factor
+        \note Multiplies the autosized capacity and flow rates
+        \type real
+        \minimum> 0.0
+        \default 1.0
+   A7 ; \field Companion Cooling Heat Pump Name
+        \type object-list
+        \object-list WWHPCoolingNames
```

Comment:
I don't think it can be figured out from context b/c these things are attached to a plant loop on both sides.  So you'd typically have the heating one connected to one plant's supply and another plant's demand, and the cooling one connected to one plant's supply and another plant's demand too.  If there were just one other of the opposite type on the same side of the plant loop then you could infer it, but if there are multiple HPs I don't think you can.  Also, if you are only using cooling or only using heating, there might not even be a companion in the model (which is still valid in E+).